packages/p2p/src/components/modal-manager/modals/my-profile-balance-modal/my-profile-balance-modal.scss

```diff
@@ -0,0 +1,9 @@
+.my-profile-balance-modal {
```

Comment:
is this style being picked up? when i had classname to the Modal wrapper, it was working only when i gave '.dc-modal__container_<classname>'

Reply:
let me double check on that

Reply:
it did apply the styles for me, but i only did it for the body and not for the entire modal